src/EditorFeatures/CSharpTest2/Recommendations/StructKeywordRecommenderTests.cs

```diff
@@ -136,6 +136,50 @@ public class StructKeywordRecommenderTests : KeywordRecommenderTests
 using Goo;");
         }
 
+        [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
+        public async Task TestAfterReadonly()
+        {
+            await VerifyAbsenceAsync(SourceCodeKind.Regular,
+@"readonly $$");
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
+        public async Task TestAfterRef()
+        {
+            await VerifyAbsenceAsync(SourceCodeKind.Regular,
+@"ref $$");
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
+        public async Task TestAfterRefReadonly()
+        {
+            await VerifyAbsenceAsync(SourceCodeKind.Regular,
+@"ref readonly $$");
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
+        public async Task TestAfterReadonlyRef()
+        {
+            await VerifyAbsenceAsync(SourceCodeKind.Regular,
+@"readonly ref $$");
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
+        public async Task TestAfterReadonlyInMethod()
+        {
+            // struct is not useful here, but offered anyways
+            await VerifyAbsenceAsync(SourceCodeKind.Regular,
```

Comment:
VerifyAbsence?

Reply:
Good catch. Thanks

Reply:
but in cases above in TestAfterReadonlyRef, TestAfterRef etc it *should* be offered right?

Reply:
Thanks